CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ STRING(TIMESTAMP TODAY "%Y%m%d")
 
 SET( PROJECT_VERSION_MAJOR 1 )
 SET( PROJECT_VERSION_MINOR 7 )
-SET( PROJECT_VERSION_PATCH "3.nonRelease.${TODAY}" )
+SET( PROJECT_VERSION_PATCH 3.nonRelease.${TODAY} )
```

Comment:
Hey, could you not keep this as a variable? It would keep changing every day, which means that we would have to recompile the entire package every day during development. Instead, let's keep the static date stamp in the form `YYYYMMDD` as we had before. Then, the change would be deliberate.